README.md

```diff
@@ -1,56 +1,116 @@
+Sample Python script written in Python 3.8.5. The script demonstrates the use of IOS-XE SLP REST APIs to automate SLP workflows.
+Smart Licensing Using Policy (SLP) was introduced in Cisco IOS-XE Amsterdam 17.3.2.
+
+
+For more information, see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
 
 ## Use Case Description
 
-For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
+For information about each API that is demonstrated in this script, see [API Endpoint and Description](ios-xe-smart-licensing-api/about-the-apis/api-endpoint-and-description).
 
 Future enhancements include an API to return a Smart Licensing Authorization Code (SLAC).
 
 ## Installation
 
-Python must be installed. Further the following Python packages are required:
-* os
-* shutil
-* json
-* xmltodict
-* requests
-* yaml
-* tarfile
+Install [Python](https://www.python.org/downloads/) and the required packages. 
+
+* Python version: 3.8.5
+* Required Python packages:
+  * os
+  * shutil
+  * json
+  * xmltodict
+  * requests
+  * yaml
+  * tarfile
+  
 
 ## Configuration
-Configure the following where? we have to check CSSM API docs to know this and then add. 
 
-client_id: 55f65e59-df52-4073-bb12-4268d5167019 (the user has to enter their own;  will enquire how this should be done)
-client_secret: 78962244-9f2f-4e50-b233-f92adedd1e33 (the user has to enter their own; will enquire how this should be done)
-username: CSSM log in username (update config.yaml with dummy username)
-password: CSSM log in password (update config.yaml with dummy password)
+
+Configure the following in the ```config.yaml``` file: 
+
+1. In the ```api_keys:``` section enter the username and password that you use to log in to the CSSM Web UI. 
+	```
+	api_keys:
+    	client_id: not required
+        client_secret: not required
+  		username: username1_cssm 
+  		password: password1_cssm
+	```
+    
+    >**Note**: Entries for ```client_id``` and ```client_secret``` are not required, because this version of the script does not interact with CSSM directly See the [Restrictions and Limitations](xe-sip-api-code/readme.md#restrictions-and-limitations) section in this README.md file for details.
+
+2. In the ```devices:``` section, enter access information for each one of your devices. The term "device" here refers to the product instance. 
+
+	For ```id:``` provide a monotonically increasing ID.
+
+	For ```ip:``` provide the management IP address
+
+	For ```username:``` provide the username to log in to the device. 
+
+	For ```password:``` provide the and the username to log in to the device. 
+
+	```
+    devices:
+  
+    	id: 1
+    	ip: 175.25.212.187
+    	username: username2
+    	password: password2
+	```
 
 
 ## Usage
 
--tbd-
+Run the Python script in a command line interface (CLI) by entering the Python command. 
```

Comment:
make it "shell prompt" or "command prompt". (CLI is for cisco devices, where you can't run python)

Reply:
Right -  thanks for catching that